bucket/synology-surveillance-station.json

```
@@ -0,0 +1,34 @@
+{
+    "version": "1.2.11-2181",
+    "architecture": {
+        "32bit": {
+            "url": "https://global.download.synology.com/download/Utility/SurveillanceStationClient/1.2.11-2181/Windows/i686/Synology%20Surveillance%20Station%20Client-1.2.11-2181_x86.zip",
+            "hash": "5bad7137b88361af8c47682de31e95126201ef4afc0964868a3acb1c75d36e89"
+        },
+        "64bit": {
+            "url": "https://global.download.synology.com/download/Utility/SurveillanceStationClient/1.2.11-2181/Windows/x86_64/Synology%20Surveillance%20Station%20Client-1.2.11-2181_x64.zip",
+            "hash": "a1530c9dc17f0a688dffdcafb757b1f5969ab7de0bfec646a3871804dda9bddf"
+        }
+    },
+    "shortcuts": [
+        [
+            "bin\\SynologySurveillanceStationClient.exe",
+            "Synology Surveillance Station Client"
+        ]
+    ],
+    "checkver": {
+        "url": "https://archive.synology.com/download/Utility/SurveillanceStationClient",
+        "regex": "<a href=\"/download/Utility/SurveillanceStationClient/(?<version>[0-9\\.-]*)\" rel=\"noreferrer noopener\">(?<displayVersion>[0-9\\.-]*)</a>",

```

```suggestion
        "regex": "/download/Utility/SurveillanceStationClient/([\\d.-]+)"
```
I think `/download/Utility/SurveillanceStationClient/...` is enough to filter out the information. There is no need for the additional `rel=....`.

---

@issaclin32 I pushed this change.